### PR TITLE
fix/sonar-issues

### PR DESCRIPTION
**Pull request by RobertGHippo**
# Description

Very minor ReSharper and Sonar issues.

# Checklist:

- [x] My code follows the standards used within this project
- [x] I have performed a self-review of my code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests (Unit, E2E) that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/Dfe.EarlyYearsQualification.Web/Controllers/QualificationDetailsController.cs [hunk 1]
```diff
@@ -34,10 +34,16 @@ public async Task<IActionResult> Get()
...
+
     [HttpPost]
     public IActionResult Refine(string refineSearch)
     {
+        if (!ModelState.IsValid)
+        {
+            logger.LogError("Invalid model state");
+            return RedirectToAction("Index", "Error");
+        }
+
```
Comment (sam-c-dfe) [inline src/Dfe.EarlyYearsQualification.Web/Controllers/QualificationDetailsController.cs:46]: On the qualification list page, this method is called when they enter something in the search box. If a user enters a value in the text box it refines the qualification list. In order to get back to the original list they can remove the search criteria and press enter. With this bit added in it redirects to the error page which we don't want as the search value is null which causes the model state to be invalid . Please can this be removed? :-)

Comment (RobertGHippo) [reply src/Dfe.EarlyYearsQualification.Web/Controllers/QualificationDetailsController.cs]: As discussed. Removed redirect, just log. Also, allowed for passing in null `refineSearch` param, which I think is how .NET will treat empty string.

File: content/Dfe.EarlyYearsQualification.ContentUpload/Program.cs [hunk 2]
```diff
@@ -15,6 +15,8 @@ public static class Program
     private const string SpaceId = "";
     private const string ManagementApiKey = "";
     private const string CsvFile = "./csv/ey-quals-full-2024-with-new-reference-fields.csv";
+    private const string QualificationContentTypeId = "Qualification";
```
Comment (sam-c-dfe) [inline content/Dfe.EarlyYearsQualification.ContentUpload/Program.cs:18]: You could potentially use Dfe.EarlyYearsQualification.Content.Constants.ContentTypes.Qualification here. Not a blocking issue though

**Review by sam-c-dfe — DISMISSED** [review]

File: content/Dfe.EarlyYearsQualification.ContentUpload/Program.cs (hunk 2 above)
Comment (RobertGHippo) [reply content/Dfe.EarlyYearsQualification.ContentUpload/Program.cs]: Good suggestion, thank you. Done.

**Review by sam-c-dfe — APPROVED** [review]

**Pull request merged**